fe/fe-core/src/main/java/com/starrocks/catalog/Table.java

```diff
@@ -387,6 +387,14 @@ public boolean isExprPartitionTable() {
         return false;
     }
 
+    public boolean isTemporaryTable() {
```

Comment:
This is not elegant, how about in base class, by default, return false, in sub-class, i.e. OlapTable, override this method.

Reply:
> This is not elegant, how about in base class, by default, return false, in sub-class, i.e. OlapTable, override this method.

fixed